buildSrc/src/main/kotlin/com/bmuschko/gradle/docker/FunctionalTestPlugin.kt

```diff
@@ -36,7 +36,8 @@ class FunctionalTestPlugin : Plugin<Project> {
                 events("started", "passed", "failed")
             }
         }
+        functionalTest.include("**/DockerCreateContainer*")
```

Comment:
Why exactly is this needed?

Reply:
It's not. I was playing around with setting `include = '...'` but it seems kotlin doesn't like that so I tried on the reference itself which seems fine. Was for dev/testing purposes. Taking out now.